manifests/VSCodium/VSCodium/1.49.0.yaml

```
@@ -0,0 +1,24 @@
+# ================== General ==================
+Id: VSCodium.VSCodium
+Publisher: VSCodium
+Name: VSCodium
+Author: VSCodium
+Description: VSCodium is a community-driven, freely-licensed binary distribution of Microsoft’s editor VSCode
+AppMoniker: vscodium
+Commands: codium
+Tags: editor, vscodium
+#FileExtensions: txt, cfg
+
+# ================== License ==================
+Homepage: https://vscodium.com/
+License: MIT License
+LicenseUrl: https://github.com/VSCodium/vscodium/blob/master/LICENSE 
+
+# ================= Installer =================
+Version: 1.49
```

```suggestion
Version: 1.49.0
```

filename and version must be identical.